src/cli/grain.js

```diff
@@ -67,6 +75,20 @@ const grainCommand: Command = async (args, std) => {
       recipientIdentities.size
     } identities in ${distributions.length} distributions`
   );
+  console.log();
```

Comment:
why?

Reply:
Formatting to differentiate the summary of our potentially many distributions from this first one that shows up.